modules/rostests/apitests/atl/CAtlList.cpp

```diff
@@ -29,18 +30,18 @@ START_TEST(CAtlList)
     ok_size_t(list1.GetCount(), 3);
 
     list1.InsertBefore(head, -123);
-    list1.InsertAfter(head, 34);    // no longer head, but the POSITION should still be valid..
+    list1.InsertAfter(head, 34); // no longer head, but the POSITION should still be valid..
 
     list1.InsertBefore(tail, 78);
     list1.InsertAfter(tail, -44);
 
-    int expected[] = {-123, 12, 34, 56, 78, 90, -44 };
+    int expected[] = {-123, 12, 34, 56, 78, 90, -44};
     int expected_size = sizeof(expected) / sizeof(expected[0]);
```

Comment:
could use `_countof` or `ARRAYSIZE` btw.

Reply:
Not really, because this file should also be usable from the .sln file that does not use any ReactOS includes.